I'm seeing additional failures:
```
error: builder for '/nix/store/qpc5pzl7dw5r7bb1qwcwpnhn9yhxkszv-python3.11-celery-5.3.6.drv' failed with exit code 1;
       last 10 log lines:
       >
       > -- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html
       > =========================== short test summary info ============================
       > [Check if "stamped_headers" is not in {}] SUBFAIL t/unit/tasks/test_stamping.py::test_canvas_stamping::test_stamping_headers_in_options[stamped_canvas-canvas_workflow11-stamping_visitor0] - AssertionError
       > [Check if "stamped_headers" is not in {}] SUBFAIL t/unit/tasks/test_stamping.py::test_canvas_stamping::test_stamping_headers_in_options[stamped_canvas-canvas_workflow12-stamping_visitor0] - AssertionError
       > [Check if "stamped_headers" is not in {}] SUBFAIL t/unit/tasks/test_stamping.py::test_canvas_stamping::test_stamping_headers_in_options[stamped_canvas-canvas_workflow13-stamping_visitor0] - AssertionError
       > [Check if "stamped_headers" is not in {}] SUBFAIL t/unit/tasks/test_stamping.py::test_canvas_stamping::test_stamping_headers_in_options[stamped_canvas-canvas_workflow3-stamping_visitor0] - AssertionError
       > FAILED t/unit/tasks/test_stamping.py::test_canvas_stamping::test_stamping_with_replace[stamped_canvas-canvas_workflow3-stamping_visitor0] - assert False
       > = 5 failed, 3024 passed, 47 skipped, 3 xfailed, 51 warnings, 25707 subtests passed in 21.90s =
```

--- a/pkgs/development/python-modules/celery/default.nix
+++ b/pkgs/development/python-modules/celery/default.nix
@@ -18,6 +18,7 @@
 , pytest-click
 , pytest-subtests
 , pytest-timeout
+, pytest-xdist
 , pytestCheckHook
 , python-dateutil
 , pythonOlder
@@ -63,6 +64,7 @@ buildPythonPackage rec {
     pytest-click
     pytest-subtests
     pytest-timeout
+    pytest-xdist
     pytestCheckHook
   ];
 
@@ -77,6 +79,10 @@ buildPythonPackage rec {
   disabledTests = [
     "msgpack"
     "test_check_privileges_no_fchown"
+    # fails with pytest-xdist
+    "test_itercapture_limit"
+    "test_stamping_headers_in_options"
+    "test_stamping_with_replace"
   ] ++ lib.optionals stdenv.isDarwin [
     # too many open files on hydra
     "test_cleanup"

